### DISCUSSION TITLE
Disable File Upload and chat Controls for users

### DISCUSSION THREAD

**Discussion by Yahya222** [open-webui/open-webui#5476] [General]
**Is your feature request related to a problem? Please describe.**
I want to allow users only to chat with the available models. I dont want to give them access to Controls and I dont want to allow them to upload the files

**Describe the solution you'd like**
Disable controls and file upload from user UI
**Describe alternatives you've considered**

**Comment by tkg61** [comment]
i would also like this feature

**Comment by hotlabs-technology** [comment]
Me too

**Comment by Hsun1128** [comment]
I think this is a good way to manage users, preventing them from mistakenly adjusting some settings.

**Comment by Koesn** [comment]
Yes, this is very needed. Using role to disable file upload still doesn't work.
